docs/_includes/js/tabs.html

```diff
@@ -28,6 +29,9 @@ <h2 id="tabs-examples">Example tabs</h2>
       <div role="tabpanel" class="tab-pane fade" id="dropdown2" aria-labelledBy="dropdown2-tab">
         <p>Trust fund seitan letterpress, keytar raw denim keffiyeh etsy art party before they sold out master cleanse gluten-free squid scenester freegan cosby sweater. Fanny pack portland seitan DIY, art party locavore wolf cliche high life echo park Austin. Cred vinyl keffiyeh DIY salvia PBR, banh mi before they sold out farm-to-table VHS viral locavore cosby sweater. Lomo wolf viral, mustache readymade thundercats keffiyeh craft beer marfa ethical. Wolf salvia freegan, sartorial keffiyeh echo park vegan.</p>
       </div>
+	  <div role="tabpanel" class="tab-pane fade" id="disabled" aria-labelledBy="disabled-tab">
```

Comment:
I think using `disabled` as the exact `id` might cause some confusion, given that there's also a `.disabled` class. Perhaps use something like `disabled-example` for the `id`?